kalite/inline/api_views.py

```diff
@@ -0,0 +1,18 @@
+"""
+Views accessible as an API endpoint for inline.  All should return JsonResponses.
+"""
+import yaml
+import json
+
+from fle_utils.internet.classes import JsonResponse
+
+
+def narrative_view(request, narrative_id):
+    f = open("kalite/inline/narratives.yaml", 'r')
+    narrative_json = yaml.safe_load(f)
+    f.close()
+
+
+    print "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!API_VIEWS!!!!!"
+    print narrative_json
+    return JsonResponse(narrative_json)
```

Comment:
Some work needs to be done here, but it depends on what you choose to use as the `pk` for narratives. The original design suggested that the keys of the narratives data would be patterns against which the url should be matched. For example, one key could be `management/zone/.*`, which would match ` `management/zone/None`,  `management/zone/YourCoolDeviceID`, and others.